packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/duration-mobile.tsx

```diff
@@ -269,10 +323,10 @@ const Duration = observer(
             return trade_basis;
         };
 
-        const onTabChange = index => {
+        const onTabChange = (index: number) => {
             setDurationTabIdx(index);
             const { value: unit } = duration_units_list[index];
-            setSelectedDuration(unit, duration_values[`${unit}_duration`]);
+            setSelectedDuration(unit, duration_values[`${unit as 't' | 's' | 'm' | 'h' | 'd'}_duration`]);
```

Comment:
can we create a type for this instead of using it inline?

Reply:
Sure, will create